kong/client.go

```diff
@@ -391,6 +455,21 @@ func (c *Client) Status(ctx context.Context) (*Status, error) {
 	return &s, nil
 }
 
+// Ready returns 200 only after the Kong node has configured itself and is ready to start proxying traffic.
```

Comment:
Can we link to the official docs here? Specifically to https://docs.konghq.com/gateway/latest/production/monitoring/healthcheck-probes/#types-of-health-checks

I'd consider mentioning that this function uses Kong's "Readiness" endpoint.